### ISSUE TITLE
Clear cache

### ISSUE THREAD

**Issue by PubliAlex** [OFFLINE-GmbH/oc-responsive-images-plugin#64]
Hello,

Is there a way to clear cached image created by responsive image plugin ?

An artisan commande like `php artisan responsiveimages:clear` would be really great

Best regards,

**Comment by tobias-kuendig** [comment]
Yes, a clear cache command would definitely be great!

In the meantime you can purge all thumbs by running the following command:

```
# Find and delete all thumb_3 files in the public temp directory.
find storage/temp/public/ -name thumb_* -delete
```

**Comment by artistro08** [comment]
That code doesn't work anymore. Is it safe to just assume the file extensions with wild cards?

**Comment by tobias-kuendig** [comment]
> That code doesn't work anymore.

What exactly does not work? In theory, you can remove all files in the *temp* directory any time as every file that it missing in it, will be re-created on demand. If you remove everything prefixed with `thumb_` you'll remove only generated thumbnails.

**Comment by artistro08** [comment]
Well, the files aren't prefixed with thumb anymore. So you can't remove them. I think it might be my own use case though. Since I'm generating the files in a different way. I did ended up finding a work around though, thank you.

**Comment by PubliAlex** [comment]
@artistro08 i need an easy way to clear image cache, what solution did you choose ?

**Comment by PubliAlex** [comment]
Just added the PR to address this issue #92

**Comment by tobias-kuendig** [comment]
Closed by #92, 86438b18829ae7be8d3235f294486347d6457c27
